lib/basket.rb

```diff
@@ -0,0 +1,17 @@
+class Basket
+  attr_reader :basket
+
+  def initialize
+    @basket = {}
+  end
+
+  def set_basket(item, quantity)
```

Comment:
I like the way you have structured this and it's easy to see what it does and how it will work, but could the method have a more clear name perhaps?

Reply:
Very good point.  I will remember to sense check method names.